prdoc/pr_3244.prdoc

```diff
@@ -0,0 +1,10 @@
+title: "Make the `benchmark pallet`` command only require a Hash"
+
+doc:
+  - audience: Node Dev
+    description: |
+      Currently the `benchmark pallet` command requires a `Block` type, whole only using its hash. Now this is changed to only require the hash. Some refactoring in DB crates was needed to achieve this.
```

Comment:
The last sentence is a little bit useless, as it doesn't bring the downstream dev any kind of information. What would be useful to mention the required changes they need to do in their code, aka switch to `HashingFor`.